### DISCUSSION TITLE
Cannot request SSL certificate (Internal Error)

### DISCUSSION THREAD

**Discussion by bbuster2** [NginxProxyManager/nginx-proxy-manager#3453] [Q&A]
I have installed nginx proxy manager on Proxmox (in a container)
Creating hosts work but as soon as i request a new SSL certificate i get an Internal Error.

I cannot find any errors in logs but it keeps crashing
How can i find what is wrong?

**Comment by iamLazyCode** [comment]
facing the same issue did you find any solution mate ?

**Comment by bbuster2** [comment]
Unfortunately not. I don't get any response from developers or users so i have no idea why i am having this or how to fix this.

**Reply by iamLazyCode** [reply]
Thanks bro. Il keep following just in case someone offers any help

**Comment by rathod-priyanka** [comment]
I also faced a similar error and just fixed it. You can resolve this issue so please follow this step-by-step guide [here](https://shorturl.at/flyMU):

**Reply by iamLazyCode** [reply]
Thankyou so much for article il check it out :)

**Reply by JupiterSnarl** [reply]
> I also faced a similar error and just fixed it. You can resolve this issue so please follow this step-by-step guide [here](https://shorturl.at/flyMU):

Doh! Link 404 not found! Help! We're all desperate for the solution! LOL

**Comment by BugZau** [comment]
I got the same things here, I hope someone can help us.

**Comment by trendmonkey** [comment]
Yeah, same here too. I also have another mailinabox server that automates letencrypt certs and that is giving me the same issue so it could be an issue with letsencrypt/certbot.
Probably going to have to try manually requesting and installing certs using certbot standalone next or fork out for annual certs from another cert authority :(

**Comment by kormpakis** [comment]
Any answers? 
I have made the same deployment on two different servers.
In the one, certs are created normally, on the other I constantly get this type of errors.

**Comment by qingyin741** [comment]
so how to solve this problem?
